### PR TITLE
added Miscellaneous class for en_US'

### PR DESCRIPTION
**Pull request by lamaiac**
added Miscellaneous class for en_US

### CODE REVIEW

File: src/Faker/Provider/en_US/Miscellaneous.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Faker\Provider\en_US;
+
+class Miscellaneous extends \Faker\Provider\Base
```
Comment (fzaninotto) [inline src/Faker/Provider/en_US/Miscellaneous.php]: You must extend `\Faker\Provider\Miscellaneous` instead

**Comment by fzaninotto** [comment]
In fact, I don't understand this PR. Miscellaneous provides random data for language, etc. Your PR implies that these formatters won't be random anymore for the en_US locale. It doesn't make sense.

File: src/Faker/Provider/en_US/Miscellaneous.php (hunk 1 above)
Comment (lamaiac) [inline src/Faker/Provider/en_US/Miscellaneous.php]: ok

**Comment by lamaiac** [comment]
@fzaninotto -- if the code specifies the locale, I want to use the locale that was specified.  As Faker sits, now, when I run this command:
*$faker = \Faker\Factory::create('en_US');*
I expect to get data for the US, but I get random country choices.

**Comment by fzaninotto** [comment]
Your `country` formatter returns "USA". You don't need Faker for that:

```php
echo 'USA';
```

Faker is a library generating random data.

**Pull request closed without merging**